### PR TITLE
Resolve sorting issues on status column for pages library

### PR DESCRIPTION
**Pull request by rkrusselenq**
## Description

The sort direction was not being set properly when clicking the Status column header.  Also adds a bit of type safety for retrieving pages.
## Tickets

* [Jira Ticket](https://cdc-nbs.atlassian.net/jira/software/c/projects/CNFT2/boards/99?assignee=712020%3A591f5469-427d-49e9-87dc-8b4df72a02b4&selectedIssue=CNFT2-1397)

## Steps to verify

1. Navigate to page-builder/manage/pages
2. Sort by the status column.
3. Verify the fetch goes from desc, to asc, and back to none.
4. Switch pagination to higher pages and verify still functions properly

### CODE REVIEW

File: apps/modernization-ui/src/apps/page-builder/generated/models/Page_PageSummary_.ts [hunk 1]
```diff
@@ -7,16 +7,15 @@ import type { PageSummary } from './PageSummary';
...
+    empty: boolean;
+    first: boolean;
+    last: boolean;
+    number: number;
+    numberOfElements: number;
+    pageable: Pageable;
+    size: number;
+    sort: Sort;
+    totalElements: number;
+    totalPages: number;
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/generated/models/Page_PageSummary_.ts:20]: Info: Was this a manual change? If so this file is an auto-generated file. We would need to add `@ApiModelProperty(required = true)` to the corresponding java object and trigger the generation.

**Comment by mpeels** [comment]
The underlying issue is due to the `Status` values in the database being stored as `IXS`, `INV`, 'CON' etc. The sort is being applied to the "Short" names which do not match the displayed names in the UI. 

I'm fairly certain Adam has resolved this issue as part of the Filter story.

File: apps/modernization-ui/src/apps/page-builder/generated/models/Page_PageSummary_.ts (hunk 1 above)
Comment (rkrusselenq) [reply apps/modernization-ui/src/apps/page-builder/generated/models/Page_PageSummary_.ts:20]: Yeah this was a manual change, thanks for the info. I just kinda figured these values would always be returned.  Should I revert or is it ok to change the @apiModelProperty to true?

**Comment by mpeels** [comment]
> > The underlying issue is due to the `Status` values in the database being stored as `IXS`, `INV`, 'CON' etc. The sort is being applied to the "Short" names which do not match the displayed names in the UI.
> > I'm fairly certain Adam has resolved this issue as part of the Filter story.
> 
> If thats the case, should we just scrap this PR and close the story?

I believe so, but let me confirm that this is fixed

**Comment by mpeels** [comment]
Confirmed sorting is resolved as part of https://github.com/CDCgov/NEDSS-Modernization/pull/688. Closing this PR

**Pull request closed without merging**